src/main/java/com/example/petbuddybackend/utils/annotation/validation/AcceptRole.java

```diff
@@ -9,6 +10,7 @@
 import java.lang.annotation.RetentionPolicy;
 import java.lang.annotation.Target;
 
+@RoleParameter
```

Comment:
aaaaaaaa, niezle
to w takim razie nie trzeba dodawac tego w endpointach wiec cofam uwagi w review xD
jakby ci sie chcialo to mozesz w OfferService przejrzec i usunac przy okazji ten RoleParameter i wgl wszedzie jak ci sie chce

Reply:
Ogarnie się